fuel/app/classes/controller/welcome.php

```diff
@@ -40,7 +40,8 @@ public function action_index()
 	 * @return  Response
 	 */
 	public function action_hello()
-	{
+  {
+    print "hogehogeffugaguga"
```

Comment:
__[development][code_sniffer]__
* Tabs must be used to indent lines; spaces are not allowed
* String "hogehogeffugaguga" does not require double quotes; use single quotes instead